feat: add beagle flutter documentation (#717)
ZupIT/beagle-docs: content/en/resources/customization/beagle-for-flutter/customization/operations.md

---
title: Custom Operations
weight: 4
description: >-

>   You will find here information of how to create and use your own operations in Beagle Flutter.

Reviewer: ```suggestion
 In this section, you will find information on how to create and use your own operations in Beagle Flutter.
```
